programs/aes/aescrypt2.c

```diff
@@ -74,7 +74,8 @@ int main( void )
 #else
 int main( int argc, char *argv[] )
 {
-    int ret = MBEDTLS_EXIT_FAILURE;
+    int retval = 0;
```

Comment:
I would prefer we left this as `ret` instead of `retval`. `ret` is idiomatic for your use case. (It would also have the benefit of making the diff far less noisy.)